spec/mock_image

```diff
@@ -0,0 +1 @@
+image data
```

Comment:
Wouldn't it be better to move this file into some subdirectory? This seems a bit out of place to me.